paltas/Configs/config_handler.py

```diff
@@ -402,14 +425,14 @@ def _draw_image_standard(self,add_noise=True,apply_psf=True):
 		single_band = SingleBand(**kwargs_detector)
 
 		# Pull the cosmology and source redshift
```

Comment:
This might be a bit an over-kill, but [here](https://github.com/sibirrer/lenstronomy/blob/e2bc2d54c14c968f7c5dddcc2b3ad4974b427113/lenstronomy/Util/class_creator.py#L14) is the lenstronomy way of initialization of the classes from the kwargs_model keyword arguments. In that way, it is a one-linear on your side and it can be updated if lenstronomy conventions would change